core/interface/views/SkillsView.py

```diff
@@ -0,0 +1,131 @@
+
+		return self.index()
+
+
+	def saveSkillSettings(self):
+		skillName = request.form['skillName']
+		for confName, confValue in request.form.items():
+			if confName == 'skillName':
+				continue
+
+			self.ConfigManager.updateSkillConfigurationFile(
+				skillName=skillName,
+				key=confName,
+				value=confValue
+			)
+
+		return self.index()
+
+
+	def updateSkill(self):
+		try:
+			_, author, skill = request.form.get('id').split('_')
+
+			self.WebInterfaceManager.newSkillInstallProcess(skill)
+			req = requests.get(f'https://raw.githubusercontent.com/project-alice-assistant/ProjectAliceSkills/{self.ConfigManager.getSkillsUpdateSource()}/PublishedSkills/{author}/{skill}/{skill}.install')
+			remoteFile = req.json()
+			if not remoteFile:
+				self.WebInterfaceManager.skillInstallProcesses[skill['skill']]['status'] = 'failed'
+
+			skillFile = Path(self.Commons.rootDir(), f'system/skillInstallTickets/{skill}.install')
+			skillFile.write_text(json.dumps(remoteFile))
+
+			return jsonify(success=True)
+		except Exception as e:
+			self.logWarning(f'Failed updating skill: {e}', printStack=True)
+			return jsonify(success=False)
+
+
+	def installSkills(self):
+		try:
+			skills = request.json
+
+			for skill in skills:
+				self.WebInterfaceManager.newSkillInstallProcess(skill['skill'])
+				req = requests.get(f'https://raw.githubusercontent.com/project-alice-assistant/ProjectAliceSkills/{self.ConfigManager.getSkillsUpdateSource()}/PublishedSkills/{skill["author"]}/{skill["skill"]}/{skill["skill"]}.install')
+				remoteFile = req.json()
+				if not remoteFile:
+					self.WebInterfaceManager.skillInstallProcesses[skill['skill']]['status'] = 'failed'
+					continue
+
+				skillFile = Path(self.Commons.rootDir(), f'system/skillInstallTickets/{skill["skill"]}.install')
+				skillFile.write_text(json.dumps(remoteFile))
+
+			return jsonify(success=True)
+		except Exception as e:
+			self.logWarning(f'Failed installing skill: {e}', printStack=True)
+			return jsonify(success=False)
+
+
+	def checkInstallStatus(self):
+		skill = request.form.get('skill')
+		status = self.WebInterfaceManager.skillInstallProcesses.get(skill, {'status': 'unknown'})['status']
+		return jsonify(status)
+
+
+	def loadStoreData(self):
+		installers = dict()
+		updateSource = self.ConfigManager.getSkillsUpdateSource()
+		req = requests.get(url=f'https://alice.maxbachmann.de/assets/{updateSource}/store/store.json')
+		results = req.json()
+
+		if not results:
+			return dict()
+
+		for skill in results:
+			if 'lang' not in skill['conditions']:
+				skill['conditions']['lang'] = constants.ALL
+			installers[skill['name']] = skill
+
+		aliceVersion = Version(constants.VERSION)
+		activeLanguage = self.LanguageManager.activeLanguage.lower()
+		return {
+			skillName: skillInfo for skillName, skillInfo in installers.items()
+			if self.SkillManager.getSkillInstance(skillName=skillName, silent=True) is None
+				and aliceVersion >= Version(skillInfo['aliceMinVersion'])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [continuation line unaligned for hanging indent](https://app.codacy.com/app/ProjectAlice/ProjectAlice/pullRequest?prid=4699682)